code/go/0chain.net/smartcontract/zcnsc/authorizer.go

```diff
@@ -77,10 +73,14 @@ func (zcn *ZCNSmartContract) AddAuthorizer(
 		return "", err
 	}
 
-	// Validating StakePoolSettings against GlobalNode settings
+	// transaction type can only be executed by sc owner
```

Comment:
The comment should be 'only sc owner can add new authorizer'. In another way to say is that this transaction will be executed only when it is created and signed by the sc owner. Transactions are executed by miners, not the sc owner.